pages/about.html

```diff
@@ -5,8 +5,37 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" type="text/css" rel="stylesheet"/> 
 </head>
-<body>
-  
+<body class="grid">
+  <header class="header header-image main-header">
+    <ul>
+      <li class="main-header-element">Mariah Rosman</li>
+      <li>she/her/hers</li>
+    </ul>
```

Comment:
I would probably recommend using an h1 tag here instead of a a ul/li block, to add the semantic meaning that this is the header.  It would also work better with the h2 block in the footer.